roadmap.md

```diff
@@ -0,0 +1,87 @@
+.NET Core Roadmap
+=================
+
+.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. The .NET Core team (anyone contributing to .NET Core) aspires to enable it to work for many interesting scenarios. Some of those are already part-way to completion. Nice work, team! 
+
+This roadmap is intended to communicate the team's priorities for evolving and extending the scope of .NET Core.
+
+Microsoft is very invested in .NET Core and makes multiple "official Microsoft" .NET Core distributions available to its customers. Sweet! It is expected (and desired) that other groups (corporations or otherwise) will also become significantly invested in .NET Core, possibly making other distros available. As a result, this roadmap is written from a neutral perspective, not a Microsoft one (unless specifically called out). This document is therefore naturally open to PRs.
```

Comment:
I think there needs to be a section somewhere that defines what exactly a "distribution" is in this case.